solarforecastarbiter/io/reference_observations/eia.py

```diff
@@ -0,0 +1,137 @@
+import os
+import logging
+from functools import partial
+import pandas as pd
+
+from solarforecastarbiter.io.fetch import eia
+from solarforecastarbiter.io.reference_observations import (
+    common, default_forecasts)
+
+from requests.exceptions import HTTPError
+
+logger = logging.getLogger('reference_data')
+
+
+def initialize_site_observations(api, site):
+    """Creates an observation at the site.
+
+    Parameters
+    ----------
+    api : solarforecastarbiter.io.api.APISession
+        API Session object, authenticated for the Reference user.
+    site : solarforecastarbiter.datamodel.Site
+        The site object for which to create the Observations.
+
+    Notes
+    -----
+    Currently only creates observations for net load [MW], but EIA contains
+    other variables that may be incorporated later (e.g. solar generation).
+
+    """
+
+    sfa_var = "net_load"
+    logger.info(f'Creating {sfa_var} at {site.name}')
+    try:
+        common.create_observation(api, site, sfa_var)
+    except HTTPError as e:
+        logger.error(f'Could not create Observation for "{sfa_var}" '
+                     f'at EIA site {site.name}')
+        logger.debug(f'Error: {e.response.text}')
+
+
+def initialize_site_forecasts(api, site):
+    """Creates a forecast at the site.
+
+    Parameters
+    ----------
+    api : solarforecastarbiter.io.api.APISession
+        API Session object, authenticated for the Reference user.
+    site : solarforecastarbiter.datamodel.Site
+        The site object for which to create the Observations.
+
+    """
+    common.create_forecasts(api, site, ["net_load"],
+                            default_forecasts.TEMPLATE_FORECASTS)
```

Comment:
This will need to be modified. `default_forecasts.TEMPLATE_DETERMINISTIC_PERSISTENCE_FORECASTS` would be an option, but I don't think we want to modify that list for a 7 day ahead forecast. So perhaps we need a `TEMPLATE_NETLOAD_PERSISTENCE_FORECASTS`.

Reply:
Adding a `TEMPLATE_NETLOAD_PERSISTENCE_FORECASTS` does make sense. I'll try adding that now.